github/path_config_test.go

```diff
@@ -181,6 +216,26 @@ func testBackendPathConfigCreateUpdate(t *testing.T, op logical.Operation) {
 		assert.Assert(t, is.Nil(resp))
 	})
 
+	t.Run("FailedStoragePersist Organization", func(t *testing.T) {
```

Comment:
You can use sub-tests for those two `FailedStoragePersist` tests, it seems only configuration differs, but not the logic.